test/Npgsql.PluginTests/JsonNetTests.cs

```diff
@@ -190,8 +195,37 @@ public class Bug3464Class
         public string? SomeString { get; set; }
     }
 
-    readonly NpgsqlDbType _npgsqlDbType;
-    readonly string _pgTypeName;
+    [Test]
+    [IssueLink("https://github.com/npgsql/npgsql/issues/4537")]
+    public async Task Write_jobject_array_without_npgsqldbtype()
+    {
+        // By default we map JObject to jsonb
+        if (!IsJsonb)
+            return;
+
+        await using var conn = await JsonDataSource.OpenConnectionAsync();
+        var tableName = await TestUtil.CreateTempTable(conn, "key SERIAL PRIMARY KEY, ingredients json[]");
```

Comment:
Why SERIAL, it's old and cruddy :rofl: Cool kids use IDENTITY... Or you can just remove and not have a primary key, no?

Reply:
That's just copy-paste from the original issue. Decided to leave it as is just in case)